governance/votes/Executive vote - October 5, 2022.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Template - [Executive Vote] Monetalis Clydesdale (RWA-007) Onboarding and initial drawdown, Funding Ambassador Program SPF, Core Unit MKR Streams and Transfer - October 5, 2022
+summary: Onboarding Monetalis Clydesdale (MIP65) as RWA-007 with a 1M DAI initial drawdown, funding transfer for the Ambassador Program Special Purpose Fund, and MKR streams and transfer to GovAlpha (GOV-001) and StarkNet Engineering (SNE-001) Core Units. 
+date: 2022-10-05T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Monetalis Clydesdale (RWA-007) Onboarding and initial drawdown, Funding Ambassador Program SPF, Core Unit MKR Streams and Transfer - October 5, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **additions** will be made to the Maker Protocol:
+- Monetalis Clydesdale (MIP65) will be onboarded as a new vault type, RWA-007, as detailed below. 
```

Comment:
```suggestion
- Monetalis Clydesdale (MIP65) will be onboarded as a new vault type, RWA007-A, as detailed below. 
```

Reply:
Not as sure on this one, as Monetalis Clydesdale is RWA-007